test/e2e/upload.test.js

```diff
@@ -228,6 +229,16 @@ describe('vl-upload', async () => {
     assert.equal(fileUploadServer.uploadedFiles.length, 0);
   });
 
+  it('als gebruiker kan ik geen bestand uploaden dat groter is dat de maximum ingestelde bestandsgrootte', async() => {
+    const upload = await vlUploadPage.getUploadMaxSize();
```

Comment:
Is deze dan nog nodig op de demo pagina? Dan kan je misschien beter testen tegen de "upload max 5" die er al stond?

Reply:
Zou kunnen. Ik zal het eens checken. Wellicht moet ik de data-vl-max-size daar wel aanpassen.

Reply:
Of je pakt een grotere file, maar die zit er nog niet in dan misschien

Reply:
zou miss ook uit de demo halen als je het hier niet gebruikt. Uiteindelijk zitten er nog andere voorbeelding in de demo die de max size zetten.

Reply:
Ik heb dit aangepast. Wordt nu ook getest tegen de upload max 5.